### DISCUSSION TITLE
Allow users to export entire published site to a PDF

### DISCUSSION THREAD

**Discussion by JaniceManwiller** [GitbookIO/community#571] [Feature Requests]
In the published content for a space, the PDF export is only available for individual topics. There isn't a way to export the entire published site to a PDF.

In GitBook itself, you can export an entire space to PDF.

It would be nice to have the option to export the entire space from the published content.

For us, each space is a user guide for a product. So it would make sense to be able to export the entire space.

**Comment by emmerich** [comment]
Hi @JaniceManwiller 

After clicking "Export to PDF", you should see a button in the bottom left to export "All Pages". This should export all pages in the space in the same PDF.

<img width="409" alt="Screenshot 2024-04-08 at 15 15 00" src="https://github.com/GitbookIO/community/assets/701222/6ab3865b-61da-4fa5-9eea-6b62f2acc431">

We export a single page by default as this is the most common use of the feature, but it might be better for us to rename this "Entire Space"

Hope this helps
